src/components/workflowEditor/Workflow.tsx

```diff
@@ -326,10 +337,30 @@ export class Workflow extends Component<WorkflowProps, WorkflowState> {
     renderWorkflow() {
         let ciPipelineId = 0
         let ciPipeline = this.props.nodes.find((nd) => nd.type == WorkflowNodeType.CI)
+        let webhookNode = this.props.nodes.find((nd) => nd.type == WorkflowNodeType.WEBHOOK)
         ciPipelineId = ciPipeline ? +ciPipeline.id : ciPipelineId
         const configDiffView = this.props.cdWorkflowList?.length > 0
 
         return (
+          <ConditionalWrap
+          condition={this.props.showWebhookTippy && !this.state.hideWebhookTippy}
+          wrap={(children) => (
+              <Tippy
+                  placement="top-start"
+                  arrow={true}
+                  trigger="manual"
+                  visible={this.props.showWebhookTippy}
+                  render={() => <WebhookTippyCard link={this.openWebhookDetails(webhookNode)} hideTippy={this.props.hideWebhookTippy}/>}
```

Comment:
Can we extract it into a separate function?

Reply:
done